fabric-datafixer-v0/src/main/java/net/fabricmc/fabric/datafixer/mixin/MixinTagHelper.java

```diff
@@ -0,0 +1,24 @@
+package net.fabricmc.fabric.datafixer.mixin;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import com.mojang.datafixers.DataFixer;
+
+import net.fabricmc.fabric.impl.datafixer.FabricDataFixerInvoker;
+import net.minecraft.datafixers.DataFixTypes;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.util.TagHelper;
+
+@Mixin(TagHelper.class)
+public class MixinTagHelper {
+    @Inject(at = @At("RETURN"), method = "update(Lcom/mojang/datafixers/DataFixer;Lnet/minecraft/datafixers/DataFixTypes;Lnet/minecraft/nbt/CompoundTag;II)Lnet/minecraft/nbt/CompoundTag;")
+    private static void updateModFixers(DataFixer dataFixer_1, DataFixTypes dataFixTypes_1, CompoundTag compoundTag_1, int dynamicDataVersion, int runtimeDataVersion, CallbackInfoReturnable<CompoundTag> cir) {
+        CompoundTag normal = cir.getReturnValue(); // We do our fixes after vanilla.
```

Comment:
Hmm, I think you capture the return values from the local vars instead of the cir

Reply:
Well TagHelper just wraps the whole method behind it's return statement and since I am applying my changes last I can just abuse the return value at end.